addons/sale/views/sale_views.xml

```diff
@@ -291,84 +320,171 @@
                                                 <field name="analytic_tag_ids" widget="many2many_tags" groups="analytic.group_analytic_tags" options="{'color_field': 'color'}"/>
                                             </div>
                                         </group>
+                                        <field name="state" invisible="1"/>
```

Comment:
Why moving it ? Reason: if someone makes an inherited view based on this field the customizatino people could do is likely to be broken. Unless necessary do not move fields in views to avoid breaking custo and inherited views.